server/journeys/referral/area-of-work/controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from 'express'
+import CsipApiService from '../../../services/csipApi/csipApiService'
+
+export class ReferralAreaOfWorkController {
+  constructor(private readonly csipApiService: CsipApiService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    req.journeyData.referral ??= {}
+
+    const areaOfWorkOptions = [
+      {
+        value: '',
+        text: 'Select area',
+        selected: !req.journeyData.referral.refererArea,
+      },
+      ...(await this.csipApiService.getReferenceData(req, 'area-of-work')).map(refData => ({
```

Comment:
Would be nice to move all of this down into the service layer so you end up with only `await this.csipApiService.getReferenceData(req, 'area-of-work', 'Select area'))` call with all the mapping and the insertion of the default option done in the service. That way that can be unit tested in isolation

Reply:
we still have use for ReferenceData[], while this function is generating govukSelect's option items which is slightly different.
i'd prefer to make it a base controller class function to convert ReferenceData[] to govukSelect options.

Reply:
As long as this code is shared somewhere, it'd fine. Base controller class function makes sense to me